DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Model/RefreshUIDSupportedVR.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Model
+{
+    public enum PerturbSupportedVR
```

Comment:
I don't think you need enum here. Maybe a hashset<string> will work for you? 
Also we can put all XXXSupportedVR into one file like 
class DicomDataModel
{
    public HashSet<string> PerturbSupportedVR;
    public HashSet<string> RefreshUidSupportedVR;
}

Reply:
Fixed it, thanks.